cmd/download.go

```diff
@@ -226,7 +226,11 @@ func buildFilename(arch, osVal string) (string, string) {
 	}
 
 	if osVal == "darwin" {
```

Comment:
I'll merge this, but I think we should somehow reference arkade and use its Tool definition / download command?